src/components/common/Posts/Post.jsx

```diff
@@ -196,10 +208,10 @@ const Post = ({props}) => {
                     <span className='fn-wg-700'>{props.user.name || "dummy name"}</span>
                         {props.user.userId !== userData.userId &&
```

Comment:
Makes sure to check this properties with null coalescing operator, it may break the app user is undefined

Reply:
ohh thanks i will put a null coalescing